runtime/taskcontext.go

```diff
@@ -55,6 +60,7 @@ type TaskContext struct {
 	webHookSet *webhookserver.WebHookSet
 	logStream  *stream.Stream
 	mu         sync.RWMutex
+	queue      *queue.Queue
```

Comment:
So I'm tempted to do: `Queue: queue.QueueInterface`

Sure someone could do `context.Queue = nil`, but that's just pure evil...